src/transformers/searchservice/createAppliedQueryFilter.ts

```diff
@@ -5,6 +5,8 @@ export function createAppliedQueryFilter(data: any): AppliedQueryFilter {
   return {
     displayKey: data.displayKey,
     displayValue: data.displayValue,
-    filter: createFilter(data.filter)
+    filter: createFilter(data.filter),
+    type: data.type,
+    ...(data.details && { details: data.details })
```

Comment:
I guess the plain `details: data.details` doesn't work here?

Reply:
is it because data.details could be a null or falsy value that's not undefined?

Reply:
if data.details is not provided by liveAPI, I thought it would be best to leave `details` field out of the returned object instead of having `details: undefined` in the object. But if that doesn't matter, I could do the plain `details: data.details`